src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -514,14 +534,30 @@ private static void getFormat(StreamType streamType, DLNAMediaInfo media, DLNAMe
 			format = FormatConfiguration.MP4;
 		} else if (value.contains("mjpg") || value.contains("m-jpeg")) {
 			format = FormatConfiguration.MJPEG;
-		} else if (value.contains("div") || value.contains("dx")) {
-			format = FormatConfiguration.DIVX;
+		} else if (
+				value.startsWith("div") ||
+				value.equals("dx50") ||
+				value.equals("dvx1")
+			) {
+				format = FormatConfiguration.DIVX;
+		} else if (value.startsWith("indeo")) { // Intel Indeo Video: IV31, IV32, IV41 and IV50
+			format = FormatConfiguration.INDEO;
+		} else if (value.equals("yuv")) {
+			format = FormatConfiguration.YUV;
+		} else if (value.equals("rgb") || value.equals("rgba")) {
```

Comment:
@Sami32 I'm not waiting for anything in particular, I just don't know how to explain it so that you understand what I mean. One major thing that I see time and time again that we think completely different about, is MediaInfo. As I see it, MediaInfo is just one of several ways to parse media, and UMS must store information in a way that makes sense regardless of what method of parsing is used. You seem to be very focused on MediaInfo as the only source of information, and use decisions made in MediaInfo as "proof" that something is correct.